build.sh

```diff
@@ -893,10 +892,6 @@ while :; do
             __PartialNgen=1
             ;;
 
-        skiptests|-skiptests)
```

Comment:
Are there any other scripts files that should be updated to remove this flags usage?

Reply:
Please update the script documentation for the `-skiptests` flag.

Reply:
Also the yml